runtests.py

```diff
@@ -20,4 +20,4 @@
         args.insert(2, 'openwisp2')
         app_dir = ['tests', 'openwisp2', 'sample_notifications']
     execute_from_command_line(args)
-    sys.exit(pytest.main([os.path.join(*app_dir, 'tests', 'test_websockets.py')]))
+    # sys.exit(pytest.main([os.path.join(*app_dir, 'tests', 'test_websockets.py')]))
```

Comment:
This is commented out. :smile:

Reply:
This is has been done deliberately due to https://github.com/openwisp/openwisp-notifications/issues/145. The build will stall if I don't comment out this code which will make it hard for us to judge this.

Reply:
Now that #145 is merged, can this line be uncommented?